Specify the PATH_CHALLENGE timer period (#3339)
quicwg/base-drafts: draft-ietf-quic-transport.md

A sender can make exceptions for probe packets so that their loss detection is
independent and does not unduly cause the congestion controller to reduce its
sending rate.  An endpoint might set a separate timer when a PATH_CHALLENGE is

> sent, which is cancelled if the corresponding PATH_RESPONSE is received or a

Reviewer: ```suggestion
sent, which is cancelled if the corresponding PATH_RESPONSE is received. If
```

> sent, which is cancelled if the corresponding PATH_RESPONSE is received or a

Reviewer: It's not clear what the "or" applies to.  You can undo the changes and simply add your new sentence at the end; see my suggestions below.

Author: Agreed, that was awkward, thanks for the suggestions.  Accepted.

> new PATH_CHALLENGE is sent if it expires before receiving a PATH_RESPONSE.

Reviewer: ```suggestion
the timer fires before the PATH_RESPONSE is received, the endpoint might send a
new PATH_CHALLENGE, and restart the timer for a longer period of time. 
```

> This timer SHOULD be set as described in section 5.3 of {{QUIC-RECOVERY}} and

Reviewer 2: ```suggestion
This timer SHOULD be set as described in Section 5.3 of {{QUIC-RECOVERY}} and
```
